arches_lingo/src/arches_lingo/components/generic/LabelViewer.vue

```diff
@@ -0,0 +1,163 @@
+<script setup lang="ts">
+import { ref } from "vue";
+import { useGettext } from "vue3-gettext";
+import DataTable from "primevue/datatable";
+import Column from "primevue/column";
+import Button from "primevue/button";
+import ConfirmDialog from "primevue/confirmdialog";
+import { useConfirm } from "primevue/useconfirm";
+
+import ControlledListItem from "@/arches_lingo/components/generic/ControlledListItem.vue";
+import ResourceInstanceRelationships from "@/arches_lingo/components/generic/ResourceInstanceRelationships.vue";
+
+import type { AppellativeStatus } from "@/arches_lingo/types";
+
+const { $gettext } = useGettext();
+const expandedRows = ref([]);
+const confirm = useConfirm();
+
+const props = defineProps<{
+    value?: AppellativeStatus[];
+}>();
+
+const emits = defineEmits(["editLabel", "deleteLabel"]);
+
+function confirmDelete(tileId: string) {
+    confirm.require({
+        header: $gettext("Confirmation"),
+        message: $gettext("Are you sure you want to delete this label?"),
+        accept: () => {
+            emits("deleteLabel", tileId);
+        },
+        rejectProps: {
+            label: "Cancel",
+            severity: "secondary",
+            outlined: true,
+        },
+        acceptProps: {
+            label: "Delete",
+            severity: "danger",
```

Comment:
the severity labels exist in constants.ts

Reply:
done in label viewer genericizing.